src/Xenial.Tasty.Cli/Commands/StudioCommand.cs

```diff
@@ -0,0 +1,188 @@
+                                    Y = 0,
+                                    Width = Dim.Fill(),
+                                    Height = Dim.Fill(),
+                                    ReadOnly = true
+                                };
+
+                                _rightPane.Add(logView);
+                                var log = string.Empty;
+                                var progress = new Progress<(string line, bool isRunning, int exitCode)>(p =>
+                                {
+                                    log += $"{p.line.Trim()}{Environment.NewLine}";
+                                    logView.Text = log;
+                                });
+
+                                var sw = Stopwatch.StartNew();
+                                var exitCode = await commander.BuildProject(path, progress, cancellationTokenSource.Token);
+                                ((IProgress<(string line, bool isRunning, int exitCode)>)progress).Report(($"Finished in {sw.Elapsed}", true, exitCode));
+                            }
+                        }
+                    }
+                    ),
+                    new MenuItem ("_Quit", "", () => Application.RequestStop() )
+                }),
+                new MenuBarItem("_Color Scheme", CreateColorSchemeMenuItems()),
+                new MenuBarItem("_About...", "About Tasty.Cli", () => MessageBox.Query("About Tasty.Cli", aboutMessage.ToString(), "_Ok")),
+            });
+
+
+
+            _statusBar = new StatusBar(new[]
+            {
+                new StatusItem(Key.ControlQ, "~CTRL-Q~ Quit", () =>
+                {
+                    Application.RequestStop();
+                }),
+                new StatusItem(Key.ControlD, "~CTRL-D~ Debug", () =>
+                {
+                    if (!Debugger.IsAttached)
+                    {
+                        Debugger.Launch();
+                    }
+                }),
+                new StatusItem(Key.ControlC, "~CTRL-C~ Cancel", () =>
+                {
+                    cancellationTokenSource.Cancel();
+                }),
+                new StatusItem(Key.F9, "~F9~ Open Menu", () =>
+                {
+                    _menu.OpenMenu();
+                }),
+            });
+
+            _top.Add(_menu, _leftPane, _rightPane, _statusBar);
+
+            SetColorScheme();
+
+            Application.Run();
+            return Task.FromResult(1);
+        }
+
+        static void SetColorScheme()
+        {
+#pragma warning disable CS8601 // Possible null reference assignment.
+            _leftPane!.ColorScheme = _baseColorScheme;
+#pragma warning restore CS8601 // Possible null reference assignment.
+            // _rightPane.ColorScheme = _baseColorScheme;
+            _top?.SetNeedsDisplay();
+        }
+
+        static ColorScheme? _baseColorScheme;
+        static MenuItem[] CreateColorSchemeMenuItems()
+        {
+            List<MenuItem> menuItems = new List<MenuItem>();
```

Comment:
Pass the Colors.ColorSchemes.Length to initialize menuItems to a known quantity.

Reply:
Done.